src/printer.ts

```diff
@@ -827,8 +831,7 @@ export class PugPrinter {
 			val = format(val, {
 				parser: 'babel',
 				...this.codeInterpolationOptions,
-				semi: useSemi,
-				endOfLine: 'lf'
```

Comment:
Did you remove `endOfLine: 'lf'` because it's already defined in `codeInterpolationOptions`?

Reply:
Yes indeed, it seemed redundant to mix it in again here?

Reply:
Yeah, I think the only real reason why this was there was a theoretical reason, cause if you don't pass `endOfLine: 'lf'` here, the formatter breaks.
And if someone want to add `endOfLine / pugEndOfLine` this will not be remembered and the suddenly break :bomb:

Reply:
Ok I'll put it back!